common/utils/src/main/resources/error/error-classes.json

```diff
@@ -2445,6 +2481,12 @@
     ],
     "sqlState" : "428C4"
   },
+  "UNRECOGNIZED_PARAMETER_NAME" : {
```

Comment:
```suggestion
  "UNRECOGNIZED_ARGUMENT_NAME" : {
```

Reply:
I asked Serge, and the original name is right. There's probably some confusion over the technical difference between argument and parameter. Typically, parameter is what each name is referring to.